examples/dns/managed-zone.gyro

```diff
@@ -0,0 +1,21 @@
+google::dns-managed-zone managed-zone-example-private
+    name: "managed-zone-example-private"
+    description: "Private Managed Zone Example"
+    dns-name: "private.example.com."
+    visibility: "private"
+
+    private-visibility-config
+        network
+            network: $(google::compute-network network-example)
```

Comment:
This network subresource should be defined within this gyro config. It should also be named contextually to this example. Something like:
```

google::compute-network network-example
    name: "managed-zone-vpc-example"
    description: "managed-zone-vpc-example-desc"
    routing-mode: "Regional"
end
```